test/integration/helpers.js

```diff
@@ -90,7 +90,8 @@ module.exports = {
           fn(
             new Error(
               format(
-                'Failed to parse JSON reporter output from result:\n\n%O',
+                'Failed to parse JSON reporter output.\nArgs: %O\nResult:\n\n%O',
+                args,
```

Comment:
Discards the `SyntaxError.message` which tells you what was wrong with the JSON, n'est-ce pas?

Reply:
does it?  not related to these changes... I'm not removing anything.

Reply:
Saying you ignored the `err.message` (from `SyntaxError` thrown by `JSON.parse()`) when handcrafting your own `Error` for display.

Reply:
oh, right, you're referring to the original code, pre-PR.  usually I don't find  JSON syntax errors to be helpful at all in this case.

it's something like "unexpected token blah at position whatever" which means ... it doesn't actually matter what it means, because execution failed and mocha output non-JSON.  so, we don't need to fix broken JSON--we need to ensure the fixture and arguments are correct.

Reply:
After having just gone through debugging with this exact code, `SyntaxError` is definitely not the only error that comes through here. The original `err.message` should be part of the new `Error` message.